src/main/resources/META-INF/plugin.xml

```diff
@@ -19,7 +19,7 @@
     ]]>
     </change-notes>
 
-    <idea-version since-build="171"/>
+    <idea-version since-build="203"/>
```

Comment:
I built it and tried to install it on Android Studio 4.1.1 (latest ATW), but I get error that it is incompatible (`Plugin 'Git Branch Cleaner' is incompatible with this installation`).

This change fixes the issue and also keeps IDEA 2020.3 working:
```suggestion
    <idea-version since-build="201"/>
```

If you want it to be compatible with older versions as well (maybe other IDEA based IDEs are even older), this number needs to be even smaller (or even left unchanged).

Reply:
thank you very much. My bad, I did not consider Android Studio.